docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+
+6. Update the `package/package.json` version with the new version number.
+
+7. Save the changes. Do not commit.
+
+8. Run `npm run build-release`, which will prompt you to either continue or cancel. Enter `y` to continue.
+
+9. If you want to make sure your changes work when used in the GOV.UK Design System, use [`npm-link`](https://docs.npmjs.com/cli/v7/commands/npm-link) to test before publishing:
+
+  ```bash
+  cd ../govuk-design-system
+  git checkout main
+  git pull
+  npm install # note running `npm install` after `npm link` will destroy the link.
+  npm link ../govuk-frontend/package/
+  ```
+
+10. When you finish testing, run `npm unlink ../govuk-frontend/package/` to unlink the package.
+
+11. Raise a pull request, with `support/<CURRENT MAJOR VERSION NUMBER>.x` as the base branch to merge into.
+
+12. Once the pull request is approved, merge it into `support/<CURRENT MAJOR VERSION NUMBER>.x`.
+
+#### Publish the release to npm
+
+1. Check out `support/<CURRENT MAJOR VERSION NUMBER>.x` and pull the latest changes.
+
+2. Run `npm login` to log into npm using our team credentials.
+
+3. Run `npm run publish-release`, which will prompt you to either continue or cancel. Enter `y` to continue.
+
+4. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases) as follows:
+
+    - select the latest tag
+    - press 'Edit tag'
+    - set ‘GOV.UK Frontend v[version-number] as the title
+    - add release notes from Changelog
+    - attach the generated ZIP that has been generated at the root of this project
+    - publish release
+
+5. Run `npm logout` to log out from npm.
+
+### After you publish the new version
+
+1. Once you've updated the GOV.UK Design System, Prototype Kit, and Frontend Docs, [post a short summary of the release in the cross-government and GDS #govuk-design-system Slack channelshttps://github.com/alphagov/govuk-frontend/issues/2363).
+
+
+2. On the [Design System Kanban Board](https://github.com/orgs/alphagov/projects/4):
+
+    - move any relevant cards from 'Ready to Release' column to 'Done'
+    - close any associated milestones
+
+## Update the `main` branch
+
+1. Check out the `main` branch and pull the latest changes.
+
+2. Run `nvm use` and `npm install` to make sure you have the latest dependencies installed.
+
+3. Make the same changes as in the patch fix PR, and test them using our [standard testing requirements](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/testing.md). Remember that `main` will contain changes the support branch did not have, which might affect the code changes you’ll need to make.
+
+4. Also update the [CHANGELOG.md](https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md) with this change. Add a new ‘Unreleased’ heading above the change, so people raising new PRs know where to add them in the Changelog. Remember that the PR links in the Changelog notes will need to link to the PRs against the `main` branch.
```

Comment:
```suggestion
4. Also update the [CHANGELOG.md](/CHANGELOG.md) with this change. Add a new ‘Unreleased’ heading above the change, so people raising new PRs know where to add them in the Changelog. Remember that the PR links in the Changelog notes will need to link to the PRs against the `main` branch.
```

Reply:
@lfdebrux I was wondering if it would be helpful here - it's important that the person following these docs is on the `main` branch and updates the Changelog on `main`. There is a step above to switch to the `main` branch, but this might help reinforce that?

Reply:
@vanitabarrett I don't think they should be editing the CHANGELOG on the main branch? I think we have branch protection rules that prevent that...

Reply:
Ah no, sorry, I meant a reminder that they need to be editing the Changelog as it is on the `main` branch rather than the one they changed via the support branch (as the two will now look different). But good point, they'll be on their own branch at this point, so it should be fine to be a relative link